fastlane/jetpack_metadata/it/subtitle.txt

```diff
@@ -1 +1 @@
-Rendi più sicuro e più rapido il tuo sito WordPress.
\ No newline at end of file
+WordPress più veloce e più sicuro
```

Comment:
33 characters ❌

I'll update in GlotPress to "WordPress più veloce e sicuro" which means the same in Italian and is 29 characters.